tests/test_model.py

```diff
@@ -1148,6 +1148,24 @@ def test_latest_filter():
     assert not ps.components.latest_components()
 
 
+@pytest.mark.django_db
+def test_released_filter():
+    sb = SoftwareBuildFactory()
+    c = ComponentFactory(software_build=sb)
+    # Make sure that a relation not of type Errata does not show in filter
+    ProductComponentRelation.objects.create(
```

Comment:
This is fine as-is, but if you want to be really thorough, we could also check that a component with no relations doesn't end up in `released_components()`. We could also test the include=False case, and make sure the filter still works when there's more than one component:
```suggestion
    c = ComponentFactory(software_build=sb)
    c2 = ComponentFactory(software_build=sb)
    # Make sure that components with no relations don't show in filter
    assert c not in Component.objects.get_queryset().released_components()
    assert c in Component.objects.get_queryset().released_components(include=False)

    # Make sure that a relation not of type Errata does not show in filter
    ProductComponentRelation.objects.create(
```